src/router/index.js

```diff
@@ -44,11 +43,6 @@ const routes = [
         name: "Orders",
         component: ordersPage
     },
-    {
```

Comment:
Why was there no need for the OrdersAddPage anymore?

Reply:
As per a related comment in a previous PR, it was a page I started that you took over, which now has a different component name and many changes. To avoid any merge conflict, I thought it was best to delete the component and just have your version replace it.